### PR TITLE
[squirrel] update to 3.2

### PR DESCRIPTION
**Pull request by MatusGuy**
- [ ] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [ ] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by MatusGuy** [comment]
@microsoft-github-policy-service agree

File: ports/squirrel/portfile.cmake [hunk 1]
```diff
@@ -15,11 +14,22 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
...
+if(BUILD_SQ)
+    set(SQ_DISABLE_INTERPRETER NO)
+else()
+    set(SQ_DISABLE_INTERPRETER YES)
+endif()
+
+message("MARTY")
+message("-DSQ_DISABLE_INTERPRETER=${SQ_DISABLE_INTERPRETER}")
+message("${FEATURE_OPTIONS}")
+
```
Comment (JonLiu1993) [inline ports/squirrel/portfile.cmake:26]:
```suggestion
```

Comment (JonLiu1993) [reply ports/squirrel/portfile.cmake:26]: I don't recommend that we keep so much debugging information here.

File: ports/squirrel/portfile.cmake [hunk 2]
```diff
@@ -15,11 +14,22 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         "interpreter" BUILD_SQ
 )
 
+if(BUILD_SQ)
```
Comment (JonLiu1993) [inline ports/squirrel/portfile.cmake:17]: @MatusGuy, Thanks for your PR, could you please clarify how to trigger the conditions here?

**Comment by dg0yt** [comment]
Maybe review waits until it is no longer a draft?

**Comment by JonLiu1993** [comment]
Pinging @MatusGuy for response. Is work still being done for this PR?

**Comment by MatusGuy** [comment]
No, it's not. Sorry.

**Comment by JonLiu1993** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**